feat(i18n): add serbian translation

cobisimo opened the pull request:
## Description

Initial translation in Serbian language. Tested in UI.

## Visual changes

Whole app is affected, so no specific screenshot.

## Functional changes

- [ x ] Code style has been fixed by running the `composer cs` command.
- [ x ] Changes have been tested to not break existing functionality.
- [ - ] Documentation is up-to-date.

cobisimo committed:
Serbian translation

cobisimo committed:
New translations

cobisimo committed:
Spelling error fixes

danharrin reviewed (changes requested):
`file-upload.js` probably needs updating and recompiling with the new language, alongside `date-time-picker.js`

danharrin reviewed (changes requested):
Please also rename to `sr_Cyrl`, which seems to be the one used by the community most often: https://github.com/Laravel-Lang/lang/tree/main/locales

cobisimo committed:
Rename serbian cyrillic directory

cobisimo committed:
Add serbian latin variant (autogenerated from cyrillic)

cobisimo commented:
- I renamed locale to match community standard and also added latin variant.
- I applied change to support date-time-picker.js (and recompiled js)
- For date-time-picker.js there is no translation yet. I created PR to their codebase: https://github.com/pqina/filepond/pull/1046

cobisimo committed:
Translation fix

danharrin committed:
Update translators.csv
